src/styles/index.scss

```diff
@@ -1,3 +1,10 @@
+@use './blocks/card.scss';
+@use './blocks/stars.scss';
+@use './blocks/header.scss';
+@use './blocks/catalog.scss';
+
 body {
   margin: 0;
+  display: flex;
+  flex-direction: column;
 }
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it is recommended to use 2 spaces for indentation instead of 4. Please adjust the indentation for consistency with the style guide.